webpack/ForemanTasks/Components/TasksDashboard/Components/TasksCardsGrid/Components/ScheduledTasksCard/ScheduledTasksCard.js

```diff
@@ -32,7 +32,12 @@ const ScheduledTasksCard = ({
     >
       <Card.Title onClick={onClick}>{__('Scheduled')}</Card.Title>
       <Card.Body>
-        <div className="scheduled-data" onClick={onClick}>
+        <div
+          className={classNames('scheduled-data', {
+            'not-focused': query.state && query.state !== SCHEDULED,
```

Comment:
I guess we don't need double checking for `query.state`, `query.state !== SCHEDULED` would be enough

Reply:
We do, it should be `not-focused` if there is something in `query.state` but it is not `SCHEDULED`.

Reply:
If `query.state` is empty, we should not add `not-focused`

Reply:
I think we are talking on the same thing, 
empty query state is not equal to `SCHEDULED` which means it will return false,
I am just suggesting writing: ` 'not-focused': query.state !== SCHEDULED,` instead.

Reply:
We are not, `not-focused` should be `false` when `query.state` is empty and `undefined !== SCHEDULED` will return `true`.

Reply:
In other words, `not-focused` shot be true only when `query.state` **has a value** and this value is different from `SCHEDULED`.
In other words, `not-focused` mean **another card is focused**